ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+#### Problem
+
+...
+
+#### Steps to replicate
+
+Provide example config and message
+
+#### Expected Behavior or What you need to ask
+
+...
+
+#### Using Fluentd and out_http plugin versions
+
+* OS version
+* Fluentd v0.12 or v0.14/v1.0
+  * paste result of ``fluentd --version`` or ``td-agent --version``
+* out_http plugin 1.x.y or 0.x.y
+  * paste boot log of fluentd or td-agent
+  * paste result of ``fluent-gem list``, ``td-agent-gem list`` or your Gemfile.lock
+* Bear Metal or Within Docker or Kubernetes or others? (optional)
```

Comment:
"Bear Metal"
![image](https://user-images.githubusercontent.com/183227/53667913-445f5400-3c27-11e9-9f6e-7131bed9d071.png)

Reply:
HAHAHAHA!

Reply:
I'm glad you like it. Please check the code to say "bare metal" to mean the following:
![My Servers](https://user-images.githubusercontent.com/183227/53672672-66150700-3c38-11e9-80fa-34e8fedb0805.jpg)